crates/wasmtime/src/config.rs

```diff
@@ -1318,6 +1314,44 @@ impl Config {
             )?)),
         }
     }
+
+    #[cfg(compiler)]
+    pub(crate) fn build_compiler(&mut self) -> Result<Box<dyn wasmtime_environ::Compiler>> {
+        let mut compiler = compiler_builder(self.compiler_config.strategy)?;
```

Comment:
If you'd like I think that this is now the only usage of the `compiler_builder` function so it should be fine to inline directly into this function.